cert-manager/overlays/moc/smaug/api/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: openshift-config
+resources:
+  - api-cert.yaml
+  - openshift-api-issuer.yaml
+  - ../../common
```

Comment:
We don't want to include `../../common` here, because that will create the `openshift-ingress` issuer, which is already created by the `smaug/ingress` directory. What problem needs to happen here is:

- Split up `common` into `secrets`, `ingress-issuer`, and `api-issuer`.
- Make the `balrog` overlay look like `smaug` and `infra` in terms of layout (e.g., have an `ingress` directory with the ingress configuration)
- Where we're creating an ingress issuer for `operate-first.cloud`, include `../../common/secrets` and `../../common/ingress-issuer`
- Where we're creating an api issuer for `operate-first.cloud` (currently just Smaug), include `../../common/api-issuer`.

Does that make sense? The goal is basically to (a) make everything look as similar as possible and (b) avoid creating objects in the wrong place.